style/layout.less

```diff
@@ -7,13 +7,11 @@ body {
     margin: 0;
 
     #root {
-        width: 100%;
         padding: 0;
         margin: 0;
 
         .page {
-            width: 100%;
-            height: @pageHeight;
+            // @todo confirm whether we need the page class name
```

Comment:
The idea was to specify the fullscreen nature of each top-level components (Search, Article, etc) here instead of individually.

If it's not needed anymore, feel free to remove it.

Reply:
Done.